kf_lib_data_ingest/etl/ingest_pipeline.py

```diff
@@ -15,7 +15,7 @@
 from kf_lib_data_ingest.etl.configuration.ingest_package_config import (
     IngestPackageConfig
 )
-from kf_lib_data_ingest.etl.configuration.log import setup_logger
+from kf_lib_data_ingest.etl.configuration.log import setup_logger, timestamp
```

Comment:
This is incorrect though not flagged as such by python because it's imported from common.misc inside configuration.log